Makefile

```diff
@@ -290,6 +290,8 @@ ifdef IMAGE_FORMAT
 image-to-cluster:
 	@echo "IMAGE_FORMAT variable detected. We're in a CI enviornment."
 	@echo "We're in a CI environment, skipping image-to-cluster target."
+	$(eval component = $(RELATED_IMAGE_OPENSCAP_NAME))
+	$(eval RELATED_IMAGE_OPENSCAP_PATH = $(IMAGE_FORMAT))
```

Comment:
FYI, If https://github.com/openshift/compliance-operator/pull/704 merges before this, IMAGE_FORMAT will need to be IMAGE_FROM_CI

Reply:
Let's merge #704 first, then I can fix this PR